### PR TITLE
LPS-96583 getting a specific entry rest query implemented

### PR DESCRIPTION
**Pull request by zoltancsaszi**

### CODE REVIEW

**Comment by zoltancsaszi** [comment]
ci:test:sf

**Comment by zoltancsaszi** [comment]
ci:test:change-tracking

File: modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/EntryResourceImpl.java [hunk 1]
```diff
@@ -27,4 +34,56 @@
 	scope = ServiceScope.PROTOTYPE, service = EntryResource.class
 )
 public class EntryResourceImpl extends BaseEntryResourceImpl {
+
+	@Override
+	public Entry getEntry(Long entryId) throws Exception {
+		CTEntry ctEntry = _ctEntryLocalService.getCTEntry(
+			GetterUtil.getLong(entryId));
```
Comment (nhpatt) [inline modules/apps/change-tracking/change-tracking-rest-impl/src/main/java/com/liferay/change/tracking/rest/internal/resource/v1_0/EntryResourceImpl.java]: why the call to GetterUtil?

**Comment by nhpatt** [comment]
Thanks! :)

**Pull request merged**